pages/admin/permit-holders.tsx

```diff
@@ -1,16 +1,344 @@
 import { GetServerSideProps } from 'next'; // Get server side props
 import { getSession } from 'next-auth/client'; // Session management
-import { Text, GridItem } from '@chakra-ui/react'; // Chakra UI
+import {
+  Box,
+  Flex,
+  Text,
+  GridItem,
+  Button,
+  Menu,
+  MenuButton,
+  InputGroup,
+  Input,
+  InputLeftElement,
+  MenuList,
+  MenuItem,
+  Wrap,
+  Badge,
+} from '@chakra-ui/react'; // Chakra UI
+import { ChevronDownIcon, SearchIcon, WarningIcon, WarningTwoIcon } from '@chakra-ui/icons'; // Chakra UI Icons
 import Layout from '@components/internal/Layout'; // Layout component
 import { Role } from '@lib/types'; // Role enum
 import { authorize } from '@tools/authorization'; // Page authorization
+import Table from '@components/internal/Table'; // Table component
+import Pagination from '@components/internal/Pagination'; // Pagination component
+import { useState, SetStateAction, Dispatch } from 'react'; // React
+import DayPicker, { DateUtils, DayPickerProps, RangeModifier } from 'react-day-picker'; // Date picker
+import 'react-day-picker/lib/style.css'; // Date picker styling
+import Helmet from 'react-helmet'; // Date picker inline styling for select range functionality
+
+// Placeholder data
+
+const DATA = Array(10).fill({
+  name: {
+    name: 'Charmaine Wang',
+    id: 36565,
+  },
+  dateOfBirth: '2021/01/01',
+  homeAddress: {
+    address: '5300, No.3 Rd Lansdowne Centre',
+    city: 'Richmond',
+    postalCode: 'V2X 1E4',
+  },
+  email: 'charmainewang@rcd.org',
+  phone: '000-000-0000',
+  recentAPP: {
+    appNumber: '#XXXXXX',
+    expiryDate: new Date(),
+  },
+  userStatus: 'active',
+});
+
+const COLUMNS = [
```

Comment:
The widths of the columns should be set according to the Figma specs:
![image](https://user-images.githubusercontent.com/60051956/127283347-3b5b6c8b-060a-4e1a-848f-3b16af16e94e.png)

It's pretty simple, you can check out Angela and I's PR, we have an example of how to do that: https://github.com/uwblueprint/richmond-centre-for-disability/pull/34/files